app/lib/routes/main_navigator/app_route.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:app/screens/home/home.dart';
+import 'package:app/screens/new_travel/new_travel_journey/test_screen_one.dart';
+import 'package:app/screens/new_travel/new_travel_journey/test_screen_two.dart';
+
+import 'package:flutter/material.dart';
+
+class AppRouteGenerator {
+  static Route<dynamic> onGenerateRoute(RouteSettings settings) {
+    print('Generating route: ${settings.name}');
+    return MaterialPageRoute(
+      settings: settings,
+      builder: (BuildContext context) {
+        switch (settings.name) {
+          case HomeRootScreen.route:
+            return HomeRootScreen();
+
+          case TestScreenOne.route:
+            return TestScreenOne();
+
+          case TestScreenTwo.route:
+            return TestScreenTwo();
+
+          default:
+            throw Exception('Invalid route: ${settings.name}');
```

Comment:
- [ ] Do exceptions crash the application or provide a pop-up at all (notify the user without looking at build output)?

Reply:
This is actually helpful during development. In production, we will have dedicated screen that will handle undefined routes